translations/en.json

```diff
@@ -12,12 +12,12 @@
   "closesNotification": "Closes notification",
   "createdRecord": "Record created",
   "createdRecordDescription": "A new encounter was created",
-  "errorDescription": "",
+  "errorDescription": "{{errorMessage}}",
```

Comment:
@usamaidrsk Does this have any impact? This is an arbitrary string that we cannot translate.

Reply:
Can we do away with the translation then!